src/components/Page/index.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 
 import MainLayout from '../MainLayout'
 import DefaultSEO from './DefaultSEO'
-import DocumentationLayout from '../Documentation/Layout'
+import DocumentationLayout from '../DocumentationLayout'
```

Comment:
We took the rest of the `Documentation` component but left `Documentation/Layout` behind and promoted it to its own standalone component.
Taking this into the theme would mean a lot more components the layout depends on would have to be pulled in, but this layout actually isn't very closely coupled with the rest of the `Documentation` component so breaking it apart works nicely.